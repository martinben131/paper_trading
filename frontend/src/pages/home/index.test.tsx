import React from "react";
import { shallow } from "enzyme";
import Home from "./index";

test("Renders home page correctly", () => {
  const component = shallow(<Home />);
  expect(component).toMatchSnapshot();
});
