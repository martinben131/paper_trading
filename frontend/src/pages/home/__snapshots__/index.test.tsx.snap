// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders home page correctly 1`] = `
<Fragment>
  <div
    className="flex-container"
  >
    <div
      className="col"
    >
      <h1
        className="title"
      >
        Stonks is a trading website that lets you experience the stock market before you jump in for good.
      </h1>
      <h2
        className="top-line"
      >
        Practice buying and selling stocks with your own predetermined budget. We use real time data from hundreds of stocks. Sign up now for free.
      </h2>
      <Blueprint3.ButtonGroup
        className="buttons"
      >
        <Blueprint3.Button
          intent="success"
          large={true}
          onClick={[Function]}
          rightIcon="arrow-right"
          text="Create an account"
        />
        <Blueprint3.Button
          intent="primary"
          large={true}
          onClick={[Function]}
          rightIcon="log-in"
          text="Login"
        />
      </Blueprint3.ButtonGroup>
    </div>
    <div
      className="image-wrapper"
    >
      <img
        alt="Stonks man"
        src="stonks.jpg"
      />
    </div>
  </div>
  <div
    className="flex-container-light"
  >
    <img
      alt="Stonks website on laptop"
      className="laptopPicture"
      src="laptop.png"
    />
    <div
      className="col"
    >
      <h1>
        Whether you are a beginner who is looking for an introduction or an advanced trader looking to innovate, practice buying and selling stocks your own way.
      </h1>
      <br />
      <h3>
        Experiment with your trades
      </h3>
      <p>
        Whether you're an advanced trader or just starting out, you can use stonks to experiment with new strategies or reinforce existing ones.
      </p>
      <br />
      <h3>
        Loaded with functionality
      </h3>
      <p>
        Contains all functionality from buying/selling, having a watchlist, being able to search for stocks, and view transaction history.
      </p>
      <br />
      <h3>
        Simple Interface
      </h3>
      <p>
        Extremely easy to use and understand for all levels.
      </p>
      <br />
      <h3>
        Finally... It's absolutely free!
      </h3>
      <p>
        No cost to you as a consumer.
      </p>
      <br />
    </div>
  </div>
  <div
    className="flex-container"
  >
    <div
      className="container"
    >
      <div
        className="secondHeading"
      >
        <h1>
          Workflow
        </h1>
      </div>
      <div
        className="row"
      >
        <div
          className="card"
        >
          <div
            className="card-header"
          >
            <h1>
              Step 1
            </h1>
          </div>
          <img
            alt="Create account buttons"
            className="cardPic"
            src="makeAccount.jpg"
          />
          <p
            className="cardText"
          >
            Create an account free of charge and login with the details that you registered with.
          </p>
        </div>
        <div
          className="card"
        >
          <div
            className="card-header"
          >
            <h1>
              Step 2
            </h1>
          </div>
          <img
            alt="List of stocks"
            className="cardPic"
            src="searchStock.jpg"
          />
          <p
            className="cardText"
          >
            Search for any of your desired stocks in the S&P500, or browse through our simple list.
          </p>
        </div>
        <div
          className="card"
        >
          <div
            className="card-header"
          >
            <h1>
              Step 3
            </h1>
          </div>
          <img
            alt="Watchlist"
            className="cardPic"
            src="watchList.jpg"
          />
          <p
            className="cardText"
          >
            Add your desired stocks to the watchlist and observe the movements
          </p>
        </div>
        <div
          className="card"
        >
          <div
            className="card-header"
          >
            <h1>
              Step 4
            </h1>
          </div>
          <img
            alt="Dashboard"
            className="cardPic"
            src="dashboard.jpg"
          />
          <p
            className="cardText"
          >
            Buy or sell your desired stock regardless of whether it is on your watchlist and watch your profits grow!
          </p>
        </div>
      </div>
    </div>
  </div>
  <p
    className="footer"
  >
    Investment Simulator - UNSW COMP3900 Project - T3 2020 - H17A - stonks
  </p>
</Fragment>
`;
